plugin/builtin/suffixtransformer/SuffixTransformer_test.go

```diff
@@ -164,3 +164,157 @@ metadata:
   name: cm
 `)
 }
+
+func TestSuffixTransformerWithDefaultFieldSpecs(t *testing.T) {
```

Comment:
Do you mind explaining how this test covers the code changes in plugin/builtin/suffixtransformer/SuffixTransformer.go? 

IIUC, without the code in SuffixTransformer.go this test still passes, right?

Reply:
It differs from the previous test by the absence of the `fieldSpecs` attribute. As per the parent commit, the absence of `fieldSpecs` would otherwise result in [an error](https://github.com/kubernetes-sigs/kustomize/blob/master/plugin/builtin/suffixtransformer/SuffixTransformer.go#L41-L43)

```go
	if p.FieldSpecs == nil {
		return errors.New("fieldSpecs is not expected to be nil")
	}
```

Reply:
Thanks!